src/PMGDQueryHandler.cc

```diff
@@ -188,7 +188,7 @@ int PMGDQueryHandler::add_node(const protobufs::AddNode &cn,
 {
     long id = cn.identifier();
     if (id >= 0 && _cached_nodes.find(id) != _cached_nodes.end()) {
-        set_response(response, PMGDCmdResponse::Error, "Reuse of _ref value\n");
+        set_response(response, PMGDCmdResponse::Error, "Reuse of _ref value");
```

Comment:
While this commit is overall necessary, I don't see a relation to the branch at all. I suggest pulling this one and the return message fixing in a separate branch for cleanup.